File: Programmers/Inho/Level01/가운데 글자 가져오기.cs
using System;
public class Solution
{
    public string solution(string s)
    {
        string answer = "";
        int len = s.Length;
        if (len % 2 != 0) // odd
            answer += s[len / 2];
        else
        {
            answer += s[len / 2 - 1];
            answer += s[len / 2];
        }
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/가운데 글자 가져오기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(string s) {
    string answer = "";
    if(s.size()%2!=0){
        answer+= s[s.size()/2];
    }else
    {
        answer= answer+ s[s.size()/2-1]+ s[s.size()/2];
        
    }
    return answer;
}